tests/integration/utils.sh

```diff
@@ -448,3 +448,17 @@ function container_inspect()
     fail "(${LINENO}): Failed to inspect the container."
   fi
 }
+
+# This function generates a temporary directory inside a specified container.
+#
+# @container  The name of the container.
+function generate_temporary_directory_in_container()
+{
+  local container="$1"
+
+  container_exec "$container" 'mktemp --directory'
+
+  if [ "$?" -ne 0 ]; then
+    fail 'Failed to create a temporary directory inside the container'
```

Comment:
If it fails, you must return an error code and check this function return in the test.